projects/demo-integrations/cypress/tests/core/dropdown/dropdown.cy.ts

```diff
@@ -18,6 +18,13 @@ describe(`Dropdown`, () => {
             cy.get(`tui-hosted-dropdown-example-1 button`).first().click().wait(1000);
             cy.matchImageSnapshot(`3`);
         });
+
+        it(`HostedDropdown custom position`, () => {
+            cy.tuiVisit(`/components/hosted-dropdown`);
+
+            cy.get(`tui-hosted-dropdown-example-5 button`).first().click().wait(1000);
+            cy.matchImageSnapshot(`4`);
+        });
     });
 
     describe(`a12y`, () => {
```

Comment:
https://github.com/Tinkoff/taiga-ui/actions/runs/4541689143/jobs/8004265227?pr=4016

```
a12y
      (Attempt 1 of 2) Esc -> Hosted Dropdown
      1) Esc -> Hosted Dropdown
  3 passing (53s)
  1 failing
  1) Dropdown
       a12y
         Esc -> Hosted Dropdown:
     CypressError: Timed out retrying after 4050ms: `cy.click()` failed because the page updated while this command was executing. Cypress tried to locate elements based on this query:
> <input#tui_interactive_4715410[304](https://github.com/Tinkoff/taiga-ui/actions/runs/4541689143/jobs/8004265227?pr=4016#step:8:305)00000.t-input.ng-untouched.ng-pristine.ng-valid>
We initially found matching element(s), but while waiting for them to become actionable, they disappeared from the page. Common situations why this happens:
  - Your JS framework re-rendered asynchronously
  - Your app code reacted to an event firing and removed the element
You can typically solve this by breaking up a chain. For example, rewrite:
> `cy.get('button').click().click()`
to
> `cy.get('button').as('btn').click()`
> `cy.get('@btn').click()`
https://on.cypress.io/element-has-detached-from-dom
      at retryActionability (http://localhost:3333/__cypress/runner/cypress_runner.js:1[306](https://github.com/Tinkoff/taiga-ui/actions/runs/4541689143/jobs/8004265227?pr=4016#step:8:307)23:82)
      at tryCatcher (http://localhost:3333/__cypress/runner/cypress_runner.js:8914:23)
      at Promise.attempt.Promise.try (http://localhost:3333/__cypress/runner/cypress_runner.js:6188:29)
      at whenStable (http://localhost:3333/__cypress/runner/cypress_runner.js:146742:65)
      at <unknown> (http://localhost:3333/__cypress/runner/cypress_runner.js:146183:14)
      at tryCatcher (http://localhost:3333/__cypress/runner/cypress_runner.js:8914:23)
      at Promise._settlePromiseFromHandler (http://localhost:3333/__cypress/runner/cypress_runner.js:6849:31)
      at Promise._settlePromise (http://localhost:3333/__cypress/runner/cypress_runner.js:6906:18)
      at Promise._settlePromise0 (http://localhost:3333/__cypress/runner/cypress_runner.js:6951:10)
      at Promise._settlePromises (http://localhost:3333/__cypress/runner/cypress_runner.js:7031:18)
      at Promise._fulfill (http://localhost:3333/__cypress/runner/cypress_runner.js:6975:18)
      at <unknown> (http://localhost:3333/__cypress/runner/cypress_runner.js:8589:46)
  From Your Spec Code:
      at Context.eval (webpack:///../testing/cypress/commands/wait-all-img-inside.command.ts:66:49)
```